controllers/users.js

```diff
@@ -315,6 +315,35 @@ const verifyUserImage = async (req, res) => {
   }
 };
 
+/**
+ * Patch Update user nickname
+ *
+ * @param req {Object} - Express request object
+ * @param res {Object} - Express response object
+ */
+
+const updateDiscordUserNickname = async (req, res) => {
```

Comment:
setUserDiscordNickname() function expects the discordId parameter to be a string. However, the value of the discordId variable is the result of the userToBeUpdated.user.discordId expression, which could be null or undefined.

Reply:
I am handling this case of `discordId` being null/undefined with this check and showing error:
 ```
if (!discordId) {
      throw new Error("user not verified");
    }
```